l10n_ro_stock_account_mrp/models/mrp_production.py

```diff
@@ -0,0 +1,51 @@
+from odoo import models
+
+from odoo.addons.mrp_account.models.mrp_production import (
+    MrpProduction as MrpAccountProduction,
+)
+
+
+def _cal_price(self, consumed_moves):
+    """Set a price unit on the finished move according to `consumed_moves`."""
+    super(MrpAccountProduction, self)._cal_price(consumed_moves)
+    work_center_cost = 0
+    finished_move = self.move_finished_ids.filtered(
+        lambda x: x.product_id == self.product_id
+        and x.state not in ("done", "cancel")
+        and x.quantity_done > 0
+    )
+    if finished_move:
+        finished_move.ensure_one()
+        for work_order in self.workorder_ids:
+            time_lines = work_order.time_ids.filtered(
+                lambda x: x.date_end and not x.cost_already_recorded
+            )
+            duration = sum(time_lines.mapped("duration"))
+            time_lines.write({"cost_already_recorded": True})
+            work_center_cost += (duration / 60.0) * work_order.workcenter_id.costs_hour
+        if finished_move.product_id.cost_method in ("fifo", "average"):
+            qty_done = finished_move.product_uom._compute_quantity(
+                finished_move.quantity_done, finished_move.product_id.uom_id
+            )
+            extra_cost = self.extra_cost * qty_done
+            finished_move.price_unit = (
+                sum(
+                    [
+                        -sum(m.stock_valuation_layer_ids.mapped("value"))
+                        for m in consumed_moves.sudo()
+                    ]
+                )
+                + work_center_cost
+                + extra_cost
+            ) / qty_done
+    return True
+
+
+MrpAccountProduction._cal_price = _cal_price
```

Comment:
aici trebuie cu patch, altfel se apeleaza super()._cal_price, si iar ajunge in mrp_account la
m.stock_valuation_layer_ids.value
si o sa dea eroare ca pot fi 2 svl-uri